lucet-runtime/lucet-runtime-internals/src/instance/execution.rs

```diff
@@ -29,8 +29,8 @@
 //! possible points of interest in an instance's lifecycle:
 //!
 //! * `Instance created`
-//!   - terminable: `false`
-//!   - execution_domain: `Guest`
+//!   - terminable: `true`
+//!   - execution_domain: `Pending`
```

Comment:
If you wanted a two line summary of this change, this is it. It would be very useful if an instance is terminable from creation onward. Before running, it will be in the `Pending` domain.

And as mentioned in the `FIXME` below, I will make a pass at updating this before merging.